.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish new Docker image to GHCR
+on:
+  push:
+    branches:
+      - master
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Build and Publish head Docker image
+        uses: VaultVulp/gp-docker-action@1.6.0
```

Comment:
Also, I'd like to suggest not using the 3rd party action but rather rely on official Docker Actions as described [in this part of documentation](https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#publishing-a-package-using-an-action).

I think that if there's a recommended way of publishing Docker images, `VaultVulp/gp-docker-action` might not be supported in the future.

Let me know if you'd like to work on that. If not, I'll take care of it 👍